ci/scripts/get-bitstream-strategy.sh

```diff
@@ -48,9 +48,11 @@ echo "Changed files after exclusions applied:"
 if git diff --exit-code --stat --name-only ${bitstream_commit} -- "${excluded_files[@]}"; then
   bitstream_strategy=cached
 else
-  bitstream_strategy=build
+  bitstream_strategy=cached
+#  bitstream_strategy=build
 fi
 
 echo
+echo "This is just for debug!!!"
```

Comment:
do you need this? hehe

(If no, I say we fix this in a follow up PR b/c right now CI is blocked no?)

Reply:
ah I see this is part of a DO NOT SUBMIT commit.

Reply:
That was merely to force the flow to use a cached bitstream, so I could test the changes. I'll rip that commit off and merge once we're ready.